WhaleSpotting/react-app/src/styles/Home.scss

```diff
@@ -6,43 +6,45 @@
     display: flex;
     flex-direction: column;
     font-family: $fontFamily;
-
+  
     .banner-image{
         height: 40vh;
         object-fit: cover;
-    }
-}
+        width: 100%;
+    }    
 
-.home-contents {
-    margin: auto;
-    width: 100%;
-}
+    .home-contents {
+        margin: 0  auto $footerHeight auto;
```

Comment:
Not sure this is needed, we should just be using auto, the footer padding is in the .home class. If we want some margin here it shouldn't be the size of footerheight

Reply:
done, however you may notice that page navigation buttons touch the footer.

Reply:
![image](https://user-images.githubusercontent.com/84720202/127865122-d5777ae1-18c7-4841-80a3-0468eb88311b.png)

Reply:
Yeah so that is why I thought you should add a some spacing at the bottom that's not the full footerheight